src/services/user.js

```diff
@@ -9,28 +9,44 @@ import storage from './storage';
 
 /**
  * @global
- * @typedef User
- * @property {int} current user's current cl&w credits
- * @property {int} required user's required cl&w credits
+ * @typedef CLWCredits
+ * @property {Number} current user's current cl&w credits
+ * @property {Number} required user's required cl&w credits
  */
 
+/**
+ * @global
+ * @typedef AttendedEvent
+ * @property {Object} CHDate Start time of the event
+ * @property {String} CHTermCD Term code of the event
+ * @property {Object} CHTime Time your ID was scanned
+ * @property {Number} Category_ID catagory of the event
+ * @property {String} Description Given discription of the event
+ * @property {String} Event_Name The generic name of the event
+ * @property {String} Event_Title Specific title of the event
+ * @property {String} Event_Type_Name Term code of the event
+ * @property {Object} Occurrences An array with the locations, start, and end times of future events
+ * @property {String} Organization Organization hosting the event
+ * @property {Number} Required Required chapel for the student
+
 /**
  * Get events attended by the user
- * @param {String} termCode code for the semester
  * @param {String} username username of the user
+ * @param {String} termCode code for the semester
  * @return {Promise.<attendedEvents>}
```

Comment:
I see that you added a `@typedef` for the type that this function returns, but you did not update the documentation for the function. Since you're returning an array of attended events, this line should look like:
```JavaScript
@return {Promise.<AttendedEvent[]>}
```